modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -0,0 +1,372 @@
+		    "cloudwatch:GetMetricData",
+		    "ec2:CreateRoute",
+		    "ec2:DeleteRoute",
+		    "ec2:CreateVpcEndpoint",
+		    "ec2:DeleteVpcEndpoints",
+		    "ec2:CreateVpcEndpointServiceConfiguration",
+		    "ec2:DeleteVpcEndpointServiceConfigurations",
+		    "ec2:DescribeVpcEndpointServiceConfigurations",
+		    "ec2:DescribeVpcEndpointServicePermissions",
+		    "ec2:DescribeVpcEndpointServices",
+		    "ec2:ModifyVpcEndpointServicePermissions"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "secretsmanager:GetSecretValue"
+            ],
+            "Resource": "*",
+            "Condition": {
+                "StringEquals": {
+                    "aws:ResourceTag/red-hat-managed": "true"
+                }
+            }
+        }
+    ]
+}
+----
+====
+
+. Prepare the policy file, by entering the following command:
++
+[source,terminal]
+----
+$ curl -o ./rosa-installer-core.json https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/sts/4.16/sts_installer_core_permission_boundary_policy.json
+----
+
+. Create the policy in AWS and gather its ARN, by entering the following command:
++
+[source,terminal]
+----
+$ aws iam create-policy \
+--policy-name rosa-core-permissions-boundary-policy \
+--policy-document file://./rosa-installer-core.json \
+--description "ROSA installer core permission boundary policy, the minimum permission set, allows BYO-VPC, disallows PrivateLink"
+----
++
+.Example output
+[source,terminal]
+----
+{
+    "Policy": {
+        "PolicyName": "rosa-core-permissions-boundary-policy",
+        "PolicyId": "<Policy ID>",
+        "Arn": "arn:aws:iam::<account ID>:policy/rosa-core-permissions-boundary-policy",
+        "Path": "/",
+        "DefaultVersionId": "v1",
+        "AttachmentCount": 0,
+        "PermissionsBoundaryUsageCount": 0,
+        "IsAttachable": true,
+        "CreateDate": "<CreateDate>",
+        "UpdateDate": "<UpdateDate>"
+    }
+}
+----
+. Add the permission boundary policy to the installer role you want to restrict, by entering the following command:
++
+[source,terminal]
+----
+$ aws iam put-role-permissions-boundary \
+--role-name ManagedOpenShift-Installer-Role \
+--permissions-boundary arn:aws:iam::<account ID>:policy/rosa-core-permissions-boundary-policy
+----
+
+. Display the installer role to validate attached policies (including permissions boundary), by entering the following command:
++
+[source,terminal]
+----
+$ aws iam get-role --role-name ManagedOpenShift-Installer-Role \
+--output text |grep PERMISSIONSBOUNDARY
+----
++
+.Example output
+[source,terminal]
+----
+PERMISSIONSBOUNDARY	arn:aws:iam::<account ID>:policy/rosa-core-permissions-boundary-policy	Policy
+----
+
+For more information, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html[Permissions boundaries for IAM entities] in the AWS documentation.
```

Comment:
Add to additonal resources section.

```suggestion
For more information, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html[Permissions boundaries for IAM entities] (AWS documentation).
```

[Guidance](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute)